service/src/main/java/tds/assessment/services/impl/AssessmentAssembler.java

```diff
@@ -0,0 +1,69 @@
+package tds.assessment.services.impl;
```

Comment:
I anticipate we will be adding additional logic to the assembler class in the future, and having all this mapping logic in the service seems like it may get a bit cluttered.